interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -30,7 +30,7 @@ angular.module('adage.analyze.sampleBin', [
 function($log, $cacheFactory, $q, Sample, Activity, Signature, SignatureSet,
 MathFuncts, errGen) {
   var SampleBin = {
-    selectedMlModel: {
+    mlModelInfo: {
```

Comment:
Great question. My goal with this PR was to keep to the original design of `sampleBin` for now but make sure the retro-fit that @dongbohu did when he created `MlModelTracker` was fully integrated.

The entire `sampleBin.js` is due for refactoring... it is quite overgrown. I know I'm punting a lot here, but I'd like to put this change into the scope of the #63-related refactoring work, which I am working to organize next (and will maybe be broken out as a separate issue there too, but put into a logical sequence that helps create a better structure for this part of the code).